### DISCUSSION TITLE
access 权限设置没效果呀

### DISCUSSION THREAD

**Discussion by liaojunhao** [umijs/umi#6213] [Q&A]
我按照官网上面的写法 在src/access.ts 这边写上，在.umirc.ts路由上面添加鉴权，没效果他，而且umi没有useAccess这个hooks
![image](https://user-images.githubusercontent.com/18565436/110057684-fc3a1580-7d9b-11eb-8998-a0f500b10d8a.png)
![image](https://user-images.githubusercontent.com/18565436/110057722-0fe57c00-7d9c-11eb-866a-62bfcdeae7df.png)

**Comment by weilun0510** [comment]
需要在 .umirc.ts plugins 中添加 '@umijs/plugins/dist/initial-state'和'@umijs/plugins/dist/access', 并开启 access: {} 和 initialState: {}
